testing/test_kerns.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 import numpy as np
 import unittest
-from .reference import referenceRbfKernel, referencePeriodicKernel
+from testing.reference import referenceRbfKernel, referencePeriodicKernel
```

Comment:
For me it fails if I try to run the test standalone as `python testing/test_kerns.py`. It works fine with nosetests. I can't really get why though.